hermit-macro/src/system.rs

```diff
@@ -60,10 +60,13 @@ fn parse_sig(sig: &Signature) -> Result<ParsedSig> {
 
 fn validate_attrs(attrs: &[Attribute]) -> Result<()> {
```

Comment:
Can you also add a check that `#[no_mangle]` must be present? Also, this macro should not add `#[no_mangle]` anymore.

Reply:
Added

Reply:
Added